driverkit/utils/scrape_and_generate

```diff
@@ -29,14 +26,15 @@ function generate_from_kernel_releases() {
 			read -r kerneldefconfig
 			pretty_echo "Generating configs for:"
 			echo "release: $release	version: $version	target: $target"
+			echo ${kerneldefconfig} > "$TMPDIR/kernelconfigdata" # pass by file to avoid exceeding the bash env size limit
 			test $DRY_RUN == "true" || \
 				make -C "$(dirname $0)/.." \
 				-e TARGET_ARCH="${ARCH}" \
 				-e TARGET_KERNEL_RELEASE="${release}" \
 				-e TARGET_KERNEL_VERSION="${version}" \
-				-e TARGET_DISTRO="${target/${target_match}/${target_replace}}" \
+				-e TARGET_DISTRO="${target}" \
 				-e TARGET_HEADERS="${kernelurls}" \
-				-e TARGET_KERNEL_DEFCONFIG="${kerneldefconfig}" \
+				-e TARGET_KERNEL_DEFCONFIG="$TMPDIR/kernelconfigdata" \
```

Comment:
Thanks @FedeDP. Could you also document this change on `TARGTE_KERNEL_DEFCONFIG` expected value?

Reply:
Uh you are right sorry!

Reply:
Mmmh `TARGTE_KERNEL_DEFCONFIG` is not a documented var because we only document filters vars!

Reply:
You're right too :-) Let's use the opportunity, WDYT? I think the variables specific to `generate` target are missing.